.github/es2k-tests.txt

```diff
@@ -0,0 +1,8 @@
+//stratum/glue/...
+//stratum/hal/lib/p4/...
+//stratum/lib/...
+//stratum/public/...
+//stratum/hal/lib/tdi:all
+//stratum/hal/lib/tdi/es2k:es2k_chassis_manager_test
```

Comment:
Im not too familiar with bazel, but why do we need to specify individual files/testcases if line 5 already has `all`?

Reply:
`//stratum/hal/lib/tdi:all` selects just the tests in `//stratum/hal/lib/tdi`.

IIRC, `//stratum/hal/lib/tdi/...` would select everything in TDI and below. We don't want to do that, though, because it would pick up the es2k and tofino tests as well (with the wrong --define target=<target> and SDE).